Index: shopify.theme.toml.example
===================================================================
diff --git a/shopify.theme.toml.example b/shopify.theme.toml.example
deleted file mode 100644
--- a/shopify.theme.toml.example	(revision 9923d65f25d4717c849317a438b88fdba50b2d80)
+++ /dev/null	(revision 9923d65f25d4717c849317a438b88fdba50b2d80)
@@ -1,13 +0,0 @@
-[environments.production]
-theme = 000000000
-store = "store.myshopify.com"
-password  = "shptka_0000"
-ignore = [
-    "config/settings_data.json",
-    "sections/*.json",
-    "templates/*.json",
-    "templates/*.*.json",
-    "templates/customers/.*.json"
-]
-#live = true
-#allow-live = true
\ No newline at end of file
